docs/book/src/user/quick-start.md

```diff
@@ -1015,6 +1073,26 @@ clusterctl generate cluster ${CLUSTER_NAME} \
     --target-namespace ${CLUSTER_NAMESPACE} | kubectl apply -f -
 ```
 
+{{#/tab }}
+{{#tab kubevirt}}
+
+First, download the cluster manifest, because we'll need to modify it a bit:
+```bash
+CAPK_VER=$(curl "https://api.github.com/repos/kubernetes-sigs/cluster-api-provider-kubevirt/releases/latest" | jq -r ."tag_name")
+curl -L https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/releases/download/${CAPK_VER}/cluster-template.yaml -O
+```
+Now, modify the service to be with the`LoadBalancer` type:
+```bash
+sed -i 's/type: ClusterIP/type: LoadBalancer/' cluster-template.yaml
```

Comment:
Would it be possible to do this change to cluster-template in kubevirt itself?

Would be easier for users vs. telling them to modify the released cluster-template

Reply:
@davidvossel - what do you think?

Reply:
OK - working on it, here: https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/pull/202